homeassistant/components/govee_light_api/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "domain": "govee_light_api",
+  "name": "Govee Lights - Local API",
+  "codeowners": ["@galorhallen"],
+  "config_flow": true,
+  "dependencies": ["network"],
+  "documentation": "https://www.home-assistant.io/integrations/govee_light_api",
+  "homekit": {},
+  "iot_class": "local_push",
+  "quality_scale": "silver",
+  "requirements": ["govee-local-api==1.3.2"],
+  "ssdp": [],
+  "zeroconf": []
```

Comment:
Please remove empty fields. Also remove the quality scale. I would prefer to have that done in a separate PR